index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet"  href="styles.css">
+
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <a class="logo" href="#">Tiago L. Marin</a>
+            <div class="menu">
+                <img src="/assets/icon-menu.png" alt="Menu Button">
+            </div>
```

Comment:
- [ **OPTIONAL** ] For good HTML semantics, I suggest you wrap your the Menu Button inside `<nav>` as it is a navigation element and it is supposed to be a l**ink** or a **button** i.e the  `<nav>` is the main container, then a sub-container `button` or a `<a>` link the the **the menu button**